core/src/main/java/com/datastax/oss/driver/internal/core/type/DefaultVectorType.java

```diff
@@ -60,7 +60,7 @@ public String getClassName() {
   @NonNull
   @Override
   public String asCql(boolean includeFrozen, boolean pretty) {
-    return String.format("'%s(%d)'", getClassName(), getDimensions());
+    return String.format("vector<%s, %d>", getElementType().asCql(true, false), getDimensions());
```

Comment:
This is already fixed in @SiyaoIsHiding 's [PR](https://github.com/apache/cassandra-java-driver/pull/1931) for JAVA-3118.  I like the addition of a unit test for it, however, so my inclination is to merge this PR and then have Jane update her PR to remove her change.

Thoughts?

Reply:
Got confirmation from @SiyaoIsHiding in separate communication that she's fine with tweaking the JAVA-3118 PR after this goes in... so we're all set to go!